client/components/index.ts

```diff
@@ -0,0 +1 @@
+export { Back } from "./Back";
```

Comment:
We don't have to care about this, but I'll leave it just as a funfact
https://twitter.com/iamakulov/status/1331551351214645251